misk-aws/src/test/kotlin/misk/jobqueue/sqs/SqsJobQueueTest.kt

```diff
@@ -10,7 +10,7 @@ import com.amazonaws.services.sqs.model.CreateQueueRequest
 import com.google.common.util.concurrent.Service
 import com.google.inject.Provides
 import com.google.inject.util.Modules
-import misk.MiskTestingServiceModule
+import misk.service.MiskTestingServiceModule
```

Comment:
why the package name change?

Reply:
I moved it into the service package to group it with `ServiceTestingModule`, could move it out of `misk.service` now that i think about it
![image](https://user-images.githubusercontent.com/4147338/51777496-365f6600-20b2-11e9-8646-be9b8935a73f.png)

Reply:
Moved `MiskTestingServiceModule ` into `misk` so no package name change.